fm.reaper.Reaper.yml

```diff
@@ -0,0 +1,82 @@
+app-id: fm.reaper.Reaper
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+tags:
+  - proprietary
+command: reaper
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0
+  - --device=all
+  - --filesystem=home
+  - --env=ALSA_CONFIG_PATH=
+
+add-extensions:
+  org.freedesktop.LinuxAudio.Plugins:
+    directory: extensions/Plugins
+    version: "22.08"
+    add-ld-path: lib
+    merge-dirs: lv2;vst;vst3
+    subdirectories: true
+    no-autodownload: true
+
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: "22.08"
+    add-ld-path: "."
+    autodelete: false
+
+modules:
+  - name: reaper
+    buildsystem: simple
+    build-commands:
+      # Install script to extract tarball
+      - install -D apply_extra /app/bin/apply_extra
+      # Install wrapper script to run application
+      - install -D reaper /app/bin/reaper
+      # Add desktop integration
+      - install -Dm644 fm.reaper.Reaper.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      # Install flatpak metadata
+      - install -Dm644 fm.reaper.Reaper.metainfo.xml /app/share/metainfo/${FLATPAK_ID}.metainfo.xml
+    sources:
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          # Extract downnloaded tarball
+          - mkdir -p export/share
+          - tar xf reaper.tar.xz -C /app/extra/export/share
+          - rm -f reaper.tar.xz
+          # Run installer script
+          - cd export/share/reaper_linux_*
+          - ./install-reaper.sh --quiet --install /app/extra
+          # Clean up temp tiles
+          - cd ../../../
+          - rm -rf /app/extra/export/share/reaper_linux_*
+      - type: extra-data
+        filename: reaper.tar.xz
+        url: https://www.reaper.fm/files/6.x/reaper664_linux_x86_64.tar.xz
+        size: 11686496
+        sha256: a60e1caa17c886d1ca5b8c7fa7f41379906a548656e836ec1c546faadf1f84d4
+        only-arches: [x86_64]
+      - type: extra-data
+        filename: reaper.tar.xz
+        url: https://www.reaper.fm/files/6.x/reaper664_linux_aarch64.tar.xz
+        size: 10470672
+        sha256: 6f3f06b6c0ba6dc7d43b4718c2f70feaa34dd9491d553e5dbfebe8b2920ea620
+        only-arches: [aarch64]
+      - type: script
+        dest-filename: reaper
+        commands:
+          - /app/extra/REAPER/reaper
+      - type: file
+        path: fm.reaper.Reaper.desktop
+      - type: file
+        path: fm.reaper.Reaper.metainfo.xml
+    post-install:
```

Comment:
you don't need a `post-install` with buildsystem `simple`. All goes into the `build-commands`